src/test/java/io/supertokens/test/multitenant/api/TestApp.java

```diff
@@ -370,4 +370,83 @@ public void testDifferentValuesForAppIdThatShouldNotWork() throws Exception {
             }
         }
     }
+
```

Comment:
modify this so that tenant creation returns 500 in case target db is not available - also check that additional storage layer exists. Also query something with this tenant and check that it fails with a 500. When we call the API again, it should succeed and future use of the tenant should succeed as well.

Reply:
Also check tghat the older storage layer is not there anymore.